Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1240,6 +1248,15 @@ void setLockdownModeEnabledGloballyForTesting(std::optional<bool> enabledForTest
 void WebProcessPool::screenPropertiesChanged()
 {
     auto screenProperties = WebCore::collectScreenProperties();
+    if (m_suppressHDR) {
+        for (auto& properties : screenProperties.screenDataMap.values()) {
+#if HAVE(SUPPORT_HDR_DISPLAY)
+            auto maxHeadroom = std::min(std::max(properties.maxEDRHeadroom * .5f, 1.5f), 2.f);
+            properties.currentEDRHeadroom = std::min(properties.currentEDRHeadroom, maxHeadroom);
+            properties.maxEDRHeadroom = std::min(properties.maxEDRHeadroom, maxHeadroom);
```

Comment:
I think this one can just write `maxHeadroom`, no min required (or do it directly in the initial line, without the temp var).